packages/dashboard/src/components/querySettingsSync/refreshRateDropDown.tsx

```diff
@@ -0,0 +1,44 @@
+import { FormField, Select } from '@cloudscape-design/components';
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { onChangeRefreshRateAction } from '~/store/actions';
+import { DashboardState } from '~/store/state';
+import { selectIntialRefreshRate } from './utils';
+
+type SelectedOption = {
+  label?: string;
+  value?: string;
+};
+
+export const RefreshRateDropDown = () => {
+  const dispatch = useDispatch();
+  const refreshRate = useSelector(
+    (state: DashboardState) =>
+      state.dashboardConfiguration.querySettings.refreshRate
+  );
+  const [selectedRefreshRate, setSelectedRefreshRate] =
+    useState<SelectedOption>(selectIntialRefreshRate(refreshRate));
+
+  return (
+    <FormField label='Refresh rate' data-testid='refresh-rate'>
+      <Select
+        selectedOption={selectedRefreshRate}
+        onChange={({ detail }) => {
+          setSelectedRefreshRate(detail.selectedOption);
+          dispatch(
+            onChangeRefreshRateAction({
+              refreshRate: parseInt(detail.selectedOption.value ?? '5000'),
+            })
+          );
+        }}
+        options={[
+          { label: '1s', value: '1000' },
+          { label: '5s', value: '5000' },
+          { label: '10s', value: '10000' },
+          { label: '1m', value: '60000' },
+          { label: '5m', value: '300000' },
+        ]}
```

Comment:
Use constants please. You have this list of numbers duplicated in `selectInitialRefreshRate`.